frontend/src/app/modules/shared/components/enitity-editor/enitity-editor.component.html

```diff
@@ -0,0 +1,3 @@
+<link href="https://unpkg.com/grapesjs/dist/css/grapes.min.css" rel="stylesheet">
```

Comment:
Move these into index.html so they do not load aaini and make it seem weird when the editor is getting loaded